### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by scarabaeus** [Schweinepriester/github-profile-achievements#323] [General]
In a new comment on this thread, add whatever public repo that you would like starred to earn the Starstruck achievement.  Star mine and I will do the same.

**https://github.com/scarabaeus/scarabaeus**

_Note: You can see who's actually starred your repo (e.g. "returned the favor") by appending `/stargazers` to the end of your repo's URL.  For example: https://github.com/scarabaeus/scarabaeus/stargazers_

**Comment by scarabaeus** [comment]
Feel free to @ me in your message -- otherwise I check this discussion board about once per day.

**Reply by scarabaeus** [reply]
@palamarchukser And done -- thank you!

**Reply by devEyosiyas** [reply]
@scarabaeus done -> https://github.com/scarabaeus/scarabaeus

Please leave a star here https://github.com/devEyosiyas/FreedomSMSBlocker thanks.

**Reply by devEyosiyas** [reply]
> @palamarchukser done -> https://github.com/palamarchukser/js-solutions/blob/master/README.md
> Please leave a star https://github.com/devEyosiyas/FreedomSMSBlocker thanks.

**Reply by scarabaeus** [reply]
> @scarabaeus done -> https://github.com/scarabaeus/scarabaeus
> 
> Please leave a star here https://github.com/devEyosiyas/FreedomSMSBlocker thanks. 

@devEyosiyas And done -- thank you!

**Reply by WHAHA-HA** [reply]
Done. https://github.com/kingsman-fullstackdev/Audiophile
thank u

**Reply by scarabaeus** [reply]
@kingsman-fullstackdev And done -- thanks!

**Reply by idontknowtoobrother** [reply]
Done Thanks <3
https://github.com/idontknowtoobrother/WordMissingGame

**Reply by aarongeo1** [reply]
@scarabaeus done -> https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by scarabaeus** [reply]
@idontknowtoobrother Done -- thank you!

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by 1RB** [reply]
@scarabaeus done ----> https://github.com/1RB/Authworria

**Reply by ArielMAJ** [reply]
@idontknowtoobrother I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Presstomsk** [reply]
Please star my repo https://github.com/Presstomsk/AviaTickets

**Reply by ArielMAJ** [reply]
@Presstomsk I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Presstomsk** [reply]
@ArielMAJ done)

**Reply by ArielMAJ** [reply]
@Presstomsk Tyvm!

**Reply by bkpecho** [reply]
@Presstomsk hello. I starred ⭐ your repo. Please star mine too:

**My repo:** https://github.com/bkpecho/bkpecho

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by Nirzak** [comment]
@scarabaeus done https://github.com/Nirzak/All-in-one-Linux-Commands

**Reply by scarabaeus** [reply]
@Nirzak And done -- thank you!

**Reply by PaulSpoerry** [reply]
> @scarabaeus done https://github.com/Nirzak/All-in-one-Linux-Commands

Starred... return favor? https://github.com/PaulSpoerry/test

**Reply by scarabaeus** [reply]
@PaulSpoerry FYI Nirzak has already hit the achievement, so he may not be quick to respond. You may want to add a new comment on this thread as opposed to replying to someone else's (for greater visibility).

**Reply by WHAHA-HA** [reply]
Done. https://github.com/kingsman-fullstackdev/Audiophile

**Reply by gogson** [reply]
@Nirzak done ! https://github.com/gogson/MidjourneyAssistant

**Reply by gogson** [reply]
@WHAHA-HA done ;) https://github.com/gogson/MidjourneyAssistant

**Reply by xLevitate** [reply]
> Comment options
> ###
> [ <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/11460645?s=64&amp;v=4"> Nirzak ](/Nirzak) [ on Aug 19](#discussioncomment-3434094)

done https://github.com/xLevitate/neptune

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by whoami-anoint** [reply]
Okay

**Reply by ArielMAJ** [reply]
@Nirzak I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by cccy0** [comment]
@scarabaeus here ~~~  thanks~    -> https://github.com/cccy0/cccy0

**Reply by scarabaeus** [reply]
@cccy0 Done! Thank you!

**Reply by gogson** [reply]
@cccy0 done ! https://github.com/gogson/MidjourneyAssistant

**Reply by xLevitate** [reply]
> @scarabaeus here ~~~ thanks~ -> https://github.com/cccy0/cccy0

done https://github.com/xLevitate/neptune

**Reply by xLevitate** [reply]
> @cccy0 done ! https://github.com/gogson/MidjourneyAssistant

done https://github.com/xLevitate/neptune

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bacca87** [comment]
starred

https://github.com/bacca87/great2

**Reply by scarabaeus** [reply]
@bacca87 Starred back -- thanks!

**Reply by WHAHA-HA** [reply]
@bacca87, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by gogson** [reply]
@bacca87 done ! https://github.com/gogson/MidjourneyAssistant

**Reply by xLevitate** [reply]
@bacca87 done https://github.com/xLevitate/neptune

**Reply by xLevitate** [reply]
@WHAHA-HA done https://github.com/xLevitate/neptune

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by TonyxSun** [comment]
@scarabaeus starred :)  -> https://github.com/TonyxSun/PressScraper

**Reply by scarabaeus** [reply]
@TonyxSun Starred back -- thanks!

**Reply by dnsamw** [reply]
@TonyxSun starred -> https://github.com/TonyxSun/PressScraper
please leave a star here -> https://github.com/dnsamw/node-yt-downloader thanks.

**Reply by WHAHA-HA** [reply]
Done. https://github.com/kingsman-fullstackdev/Audiophile

**Reply by gogson** [reply]
@TonyxSun done ! https://github.com/gogson/MidjourneyAssistant

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
@TonyxSun I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@dnsamw I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by dnsamw** [comment]
@scarabaeus done -> https://github.com/scarabaeus/scarabaeus
please leave a star here -> https://github.com/dnsamw/node-yt-downloader thanks.

**Reply by scarabaeus** [reply]
@dnsamw Done -- thanks!

**Reply by hitblast** [reply]
@dnsamw Starred!

https://github.com/IgKniteDev/IgKnite Thanks in advance.

**Reply by dnsamw** [reply]
@hitblast Done - thanks!

**Reply by aziz0x48** [reply]
done guys, please star https://github.com/c99tn/vMass

**Reply by dnsamw** [reply]
@c99tn done - thanks

**Reply by WHAHA-HA** [reply]
@dnsamw Done 
please leave star here https://github.com/kingsman-fullstackdev/Audiophile

**Reply by dnsamw** [reply]
@kingsman-fullstackdev done-- thanks 😃

**Reply by gogson** [reply]
@dnsamw done ! https://github.com/gogson/MidjourneyAssistant

**Reply by dnsamw** [reply]
> @dnsamw done ! https://github.com/gogson/MidjourneyAssistant

Done! Thanks

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by hitblast** [comment]
@scarabaeus Done.

https://github.com/IgKniteDev/IgKnite Thank you :D

**Reply by aziz0x48** [reply]
done https://github.com/c99tn/vMass

**Reply by hitblast** [reply]
Done as well :D

**Reply by scarabaeus** [reply]
> https://github.com/IgKniteDev/IgKnite

@hitblast And done -- thank you!

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by aziz0x48** [comment]
https://github.com/c99tn/vMass done! for other people comment below I ll star !

**Reply by scarabaeus** [reply]
@c99tn And done -- thank you!

**Reply by gogson** [reply]
@c99tn done ! 
https://github.com/gogson/MidjourneyAssistant

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Ashu-Barnwal** [comment]
DONE 
https://github.com/Ashu-Barnwal/Color-Plate
thank you!!!

**Reply by hitblast** [reply]
@Ashu-Barnwal Done as well :D
https://github.com/IgKniteDev/IgKnite thank you in advance.

**Reply by aziz0x48** [reply]
@Ashu-Barnwal  https://github.com/c99tn/vMass done mate, thanks !

**Reply by scarabaeus** [reply]
@Ashu-Barnwal Done -- thank you!

**Reply by WHAHA-HA** [reply]
@Ashu-Barnwal, done
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by gogson** [reply]
@Ashu-Barnwal done !
https://github.com/gogson/MidjourneyAssistant

**Reply by Ashu-Barnwal** [reply]
@gogson DONE!

**Reply by bingyun123** [reply]
Done
https://github.com/bingyun123/custom_chart_viz

**Reply by ArielMAJ** [reply]
@bingyun123 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by scarabaeus** [reply]
@danielr1996  Done -- thank you!

**Reply by gogson** [reply]
@danielr1996 done ! 
Star please ? https://github.com/gogson/MidjourneyAssistant

**Reply by dnsamw** [reply]
@danielr1996 Done! thanks

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by kshitij-halankar** [comment]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by gogson** [reply]
Star please?
https://github.com/gogson/MidjourneyAssistant

**Reply by kshitij-halankar** [reply]
@gogson done

**Reply by scarabaeus** [reply]
@kshitij-halankar Done -- thank you!

**Reply by bingyun123** [reply]
Done
https://github.com/bingyun123/custom_chart_viz @kshitij-halankar

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ghost** [comment]


**Reply by gogson** [reply]
@AlekseevDanil done ! https://github.com/gogson/MidjourneyAssistant

**Reply by xommmax** [reply]
Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by bingyun123** [reply]
Done
https://github.com/bingyun123/custom_chart_viz

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by HHNebula** [comment]
done
https://github.com/HHNebula/HHNebula

**Reply by scarabaeus** [reply]
@HHNebula [And done](https://github.com/HHNebula/HHNebula/stargazers) Thank you!

**Reply by ArielMAJ** [reply]
@HHNebula I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by PaulSpoerry** [comment]
done: https://github.com/PaulSpoerry/test

**Reply by scarabaeus** [reply]
@PaulSpoerry Which one is you?

**Reply by PaulSpoerry** [reply]
![Screenshot_20221008-140446.png](https://user-images.githubusercontent.com/1054579/194723540-4ac9b0d0-1f1e-4431-8d72-d74811448ed7.png)

Sorry on mobile so this kinda sucks but it's my full name with no spaces.

**Reply by scarabaeus** [reply]
@PaulSpoerry Done -- thanks!

**Reply by PaulSpoerry** [reply]
🍻 cheers and thanks!

**Reply by bingyun123** [reply]
Done
https://github.com/bingyun123/custom_chart_viz

**Reply by ArielMAJ** [reply]
@PaulSpoerry I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@bingyun123 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Reply by scarabaeus** [reply]
@41pha1 And [done](https://github.com/41pha1/MIDI-Converter/stargazers) -- thank you!

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd

**Reply by scarabaeus** [reply]
@drk1rd  And done -- thank you! https://github.com/drk1rd/drk1rd/stargazers

**Reply by ArielMAJ** [reply]
@drk1rd I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bm4578** [reply]
> 完成！
> https://github.com/drk1rd/drk1rd

Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by kensoz** [comment]


**Reply by scarabaeus** [reply]
@kensoz Done -- thanks!

**Reply by ArielMAJ** [reply]
@kensoz I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by montasim** [comment]
Done @scarabaeus. Please do it for me.
https://github.com/montasim/montasim

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by hasibul442** [comment]
https://github.com/hasibul442/Online-Crime-Report-System

Please Give Star I will send back

**Reply by kensoz** [reply]
I did, would you star me back :)
@hasibul442 https://github.com/kensoz/FEC

**Reply by montasim** [reply]
Done @hasibul442. Please do it for me.
https://github.com/montasim/montasim

**Reply by hasibul442** [reply]
@kensoz @montasim  done

**Reply by hitblast** [reply]
@Buedenbender @kensoz @montasim Done!

https://github.com/IgKniteDev/IgKnite

**Reply by kensoz** [reply]
@hitblast 
Thanks. I did

**Reply by ArielMAJ** [reply]
@hasibul442 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by abhisheks008** [comment]
Star this repo, https://github.com/abhisheks008/DL-Simplified

**Reply by kensoz** [reply]
@abhisheks008 
I did, would you star me back :)
https://github.com/kensoz/FEC

**Reply by ArielMAJ** [reply]
@abhisheks008 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by haithamaouati** [comment]
Done! Could I get Galaxy Brain?

**Comment by dhrubasaha08** [comment]
My Repo https://github.com/dhrubasaha08/vanilla-minecraft-server-management
@scarabaeus Done

**Reply by hitblast** [reply]
@dhrubasaha08 Done! https://github.com/IgKniteDev/IgKnite

**Reply by dhrubasaha08** [reply]
@hitblast @Buedenbender  @Zoltus done

**Reply by kensoz** [reply]
@dhrubasaha08 
I did !
https://github.com/kensoz/FEC

**Reply by dhrubasaha08** [reply]
@kensoz done ! and thank you

**Reply by 1RB** [reply]
done 
https://github.com/1RB/Authworria

**Reply by dhrubasaha08** [reply]
@1RB done and thanks

**Reply by hitblast** [reply]
@1RB Done as well! https://github.com/IgKniteDev/IgKnite

**Reply by 1RB** [reply]
> @1RB Done as well! https://github.com/IgKniteDev/IgKnite

thank you, returned the favour 💫

**Reply by ArielMAJ** [reply]
@dhrubasaha08 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by 1RB** [comment]
star and ill star your repo back [link it below]
* https://github.com/1RB/Authworria

**Reply by 1RB** [reply]
> @1RB I did ! https://github.com/kensoz/FEC

thank you, returned the favour

**Reply by 1RB** [reply]
> Done 😄 https://github.com/Buedenbender/datscience

congrats i was the 16th starer, also thank you for staring my rep as well

**Reply by hasibul442** [reply]
Done
Here is mine https://github.com/hasibul442/Online-Crime-Report-System

**Reply by imomoe233** [reply]
done！
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor

**Reply by ArielMAJ** [reply]
@1RB, @mmwan101010, @Buedenbender I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bm4578** [reply]
@1RB Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by hitblast** [comment]
I'll repost mine :D

Star and I'll star you back.
https://github.com/IgKniteDev/IgKnite

**Reply by imomoe233** [reply]
DONE！
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor

**Reply by bm4578** [reply]
> 我会重新发布我的:D
> 
> 加星，我会加星给你。 https://github.com/IgKniteDev/IgKnite

Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by hasibul442** [comment]
star and ill star your repo back [link it below]
https://github.com/hasibul442/Online-Crime-Report-System

**Reply by hitblast** [reply]
@hasibul442 Done! https://github.com/IgKniteDev/IgKnite

**Reply by imomoe233** [reply]
DONE！
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor

**Reply by bm4578** [reply]
> star 和 ill star 你的 repo 返回 [链接在下面] 
> https://github.com/hasibul442/Online-Crime-Report-System

Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Comment by muhammadnurulahsan** [comment]
star and ill star your repo back [link it below]
https://github.com/muhammadnurulahsan/Carpenco-Store-Client-Side
https://github.com/muhammadnurulahsan/Carpenco-Store-Server-Side

**Reply by imomoe233** [reply]
DONE！
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor

**Reply by ArielMAJ** [reply]
@muhammadnurulahsan I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bm4578** [reply]
@muhammadnurulahsan Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Comment by imomoe233** [comment]
star and ill star your repo back [link it below]
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor

**Reply by bm4578** [reply]
@mmwan101010 Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bstummer** [comment]
Star and I'll star your repo back
https://github.com/bstummer/openskill.lua

@scarabaeus

**Reply by bm4578** [reply]
> 加星，我会加星你的回购 https://github.com/bstummer/openskill.lua
> 
> @scarabaeus

Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by DushanSenadheera** [comment]
Hello! can anyone help me to achieve starstruck achievement by giving some starts to this repo https://github.com/DushanSenadheera/Algorithms-and-Data-Structures

**Reply by bm4578** [reply]
@DushanSenadheera Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bm4578** [comment]
Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by montasim** [reply]
Done @bm4578

**Reply by bm4578** [reply]
thanks

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by GameExplorer** [comment]
### Star 4 Star

my repo: https://github.com/GameExplorer/GameExplorer

**Reply by bm4578** [reply]
@GameExplorer Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by GameExplorer** [reply]
Hello sure

**Reply by bm4578** [reply]
thanks

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by PannDev** [comment]
⭐ Star 4 Star ⭐
(https://github.com/PannDev/panndev.github.io)

**Reply by bm4578** [reply]
Done ! Can you help me? https://github.com/bm4578/docker.git

**Reply by PannDev** [reply]
> Done ! Can you help me? https://github.com/bm4578/docker.git

done

**Reply by ArielMAJ** [reply]
@PannDev I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by PannDev** [reply]
> @PannDev I starred you, please star my repo too: https://github.com/ArielMAJ/test.me Feel free to contribute as well, there are many somewhat "simple" issues.

done !

**Reply by hitblast** [reply]
@mmanosalva Starred your repo!

https://github.com/IgKniteDev/IgKnite

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Reply by ArielMAJ** [reply]
@bensonarafat I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by PannDev** [reply]
starback https://github.com/PannDev/panndev.github.io

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ArielMAJ** [comment]
@scarabaeus I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by hitblast** [reply]
Done! https://github.com/IgKniteDev/IgKnite

**Reply by ArielMAJ** [reply]
@mmanosalva I starred you back!

Congratz on your Starstruck badge!

**Reply by ArielMAJ** [reply]
@hitblast Ty! Already starred you!

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ittid** [comment]
I starred you please star my repo too: https://github.com/ittid/ittiddev.com

**Reply by ArielMAJ** [reply]
@ittid I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Reply by hitblast** [reply]
@4lfant Thanks for the star back there! Starring your repository. :D

**Comment by om3ga6400** [comment]
@scarabaeus https://github.com/OMEGA6405/OMEGA6405

**Reply by ArielMAJ** [reply]
@OMEGA6405 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Rantoryu** [comment]
@scarabaeus 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Comment by Presstomsk** [comment]
Ваше письмо поступило в обработку. С Вами обязательно свяжутся.

**Reply by ArielMAJ** [reply]
What, what letter?
